docs/attributes-dictionary/http.md

```diff
@@ -0,0 +1,52 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# HTTP
+
+<!-- semconv registry.http -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
```

Comment:
You have "Requirement Level" here, although above you state it's not part of the registry, but is added by semantic conventions:

> In addition, semantic conventions specify the requirement level of an attribute in the corresponding context.

At least, that's how I understood it.

Reply:
Yes, you're right, it's a current limitation of the table-generation tool.

I already [created a PR in the build-tools](https://github.com/open-telemetry/build-tools/pull/190) repo to allow generating tables without the `Requirement Level` column.

Reply:
Wondering if we should focus on getting https://github.com/open-telemetry/build-tools/pull/191 merged before this PR?

Reply:
Yeah I think the more pragmatic approach would be to add this option in the tool (even if temporally) to get the proper table rendered in the spec. Then we can just remove the param on the build-tool and implement the full-blown yaml.